src/h5web/dimension-mapper/SlicingSlider.tsx

```diff
@@ -15,7 +16,14 @@ interface Props {
 
 function SlicingSlider(props: Props) {
   const { dimension, slicingIndex, rawDims, mapperState, onChange } = props;
+
   const [containerRef, { height }] = useMeasure();
+  const sliderRef = useRef<Component<ReactSlider.ReactSliderProps>>(null);
+
+  useEffect(() => {
+    // @ts-ignore
+    sliderRef.current?.resize();
+  }, [height]);
```

Comment:
The fix for the resizing.

Reply:
Why the `// @ts-ignore` ?

Reply:
That's because the types of ReactSlider don't cover this use case of getting a reference of the slider instance and calling methods on it.